src/settings.py

```diff
@@ -11,3 +12,5 @@ class Settings(BaseSettings):
     snyk_package_url_format: HttpUrl = 'https://snyk.io/vuln/{ecosystem}:{package}'
     snyk_signin_url: HttpUrl = 'https://snyk.io/login'
     snyk_ecosystem_map: Dict[str, str] = {"pypi": "pip"}
+
+    disable_unknown_package_flow: bool = os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW", "") == "1"
```

Comment:
Pydantic settings takes care of reading the value from env variable. Please look at the existing flag(snyk_package_url_format) and do accordingly.
```suggestion
    disable_unknown_package_flow: bool = False
```